index.html

```diff
@@ -1,27 +1,332 @@
+          <div class="badge av-badge-xs"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-sm" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-sm"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-md" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-md"></div>
+        </div>
+        <div class="avatar-with-badge">
+          <img class="avatar avatar-lg" src="https://picsum.photos/200/200" />
+          <div class="badge av-badge-lg"></div>
+        </div>
+
+
+
+
+      </div>
+    </section>
+
+
+    <section>
+      <h2>Cards</h2>
+      <div class='examples'>
+        <div class="card text-only-card">
+          <div class="card-body">
+            <div class="text">
+              <div class="titleText">Our Changing Planet</div>
+              <div class="authorText">by Kurt Wagner</div>
+            </div>
+            <div class="description">
+              Visit ten places on our planet that are undergoing the biggest changes
+              today Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Adipisci quisquam suscipit magni dicta, repellat, quod illo harum
+              libero esse quidem veniam dolor dolores! Optio rerum atque maxime vero
+              at voluptatibus!
+            </div>
+          </div>
+    
+          <div class="foot">
+            <button class='btn secondary-btn'>READ</button>
+            <button class='btn secondary-btn'>BOOKMARK</button>
+            <span class="material-icons">favorite</span>
+            <span class="material-icons">share</span>
+            <span class="material-icons">more_vert</span>
+          </div>
+        </div>
+    
+        <div class="card card-with-dismiss">
+          <div class="card-body">
+            <div class="figure-with-dismiss">
+              <img src="https://picsum.photos/400/300" />
+    
+              <div class="close-btn">
+                <span class="material-icons">close</span>
+              </div>
+            </div>
+            <div class="text">
+              <div class="titleText">Our Changing Planet</div>
+              <div class="authorText">by Kurt Wagner</div>
+            </div>
+            <div class="description">
+              Visit ten places on our planet that are undergoing the biggest changes
+              today
+            </div>
+          </div>
+    
+          <div class="foot">
+            <button class='btn secondary-btn'>READ</button>
+            <button class='btn secondary-btn'>BOOKMARK</button>
+            <span class="material-icons">favorite</span>
+            <span class="material-icons">share</span>
+            <span class="material-icons">more_vert</span>
+          </div>
+        </div>
+    
+        <div class="card card-with-badge">
+          <div class="card-body">
```

Comment:
card class names can be made consistent, at some places camelCases are used and at some places "-" are used

Reply:
for example titleText

Reply:
changed to "-" notation everywhere. Thanks